### ISSUE TITLE
Crash due to unsafe blockstate property application

### ISSUE THREAD

**Issue by BiakCeu** [xyroc/DungeonCrawl#158]
Never had this type of crash before, let alone a crash with this dungeoncrawl. 
[latest.log](https://github.com/user-attachments/files/17235578/latest.log)

**Comment by xyroc** [comment]
It looks like a custom theme might be triggering the problem because of a missing check before applying block state properties.
Is the modpack you are using available somewhere?

**Comment by BiakCeu** [comment]
I don't fully understand your question but if you mean if the modpack is public, then no it isn't. Could you elaborate on what you mean by "custom theme"? You mean your themes for every 
floor in the dungeons?

**Comment by xyroc** [comment]
Yes, i mean those themes. What happened with the crash is that the mod tried to place mossy cobblestone stairs which are facing downwards, which is impossible since stairs can only have horizontal directions, not upwards or downwards.
The mod is supposed to check whether a such a block property can be applied before using it, but it clearly did not in your case.
Since i haven't seen this crash before, i am suspecting that some custom theme for a floor of the dungeon was in use, since those decide which blocks will be placed, for example the mossy cobblestone stairs.

Also, i'm wondering if you could give me the debug version of the log you posted (`debug.log`)? It has more info and could help pinpoint the problem.
